src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/__tests__/MethodologySummaryPage.test.tsx

```diff
@@ -43,9 +43,6 @@ describe('MethodologySummaryPage', () => {
         'Test methodology',
       );
 
-      expect(screen.getByTestId('Status-key')).toHaveTextContent('Status');
```

Comment:
Not saying we need to change this with this PR, but I do find these test ids a little gross to look at. 

Would probably be better if we had them as like 'Status - key/value', 'Published on - key/value', etc. Probably one for another day anyway.